packages/discovery-react-components/src/components/DocumentPreview/DocumentPreview.tsx

```diff
@@ -46,6 +46,10 @@ interface Props extends WithErrorBoundaryProps {
    * `document.document_passages` or `searchResults.table_results`.
    */
   highlight?: QueryResultPassage | QueryTableResult;
+  /**
+   * disable textLayer if true
+   */
+  disableTextLayer?: boolean;
```

Comment:
- Specify that the default is `false`
- This property is at the `DocumentPreview` level, which can render a document in several different forms. But this property only applies to `PdfViewer`. Given that, we should be more specific about its intent.
- Follow convention for these comments of using sentence case
 
So something like this:
```suggestion
  /**
   * Disable the text layer overlay when rendering PDF (defaults to `false`)
   */
  disablePdfTextLayer?: boolean;
```